From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Mon, 12 Feb 2018 21:37:52 +0100
Subject: ungoogled-chromium: Disable Gaia

Disables Gaia code.
Somehow it is still being activated even without being signed-in.

See also: https://github.com/Eloston/ungoogled-chromium/issues/104
---
 chrome/browser/browser_resources.grd              | 1 -
 chrome/browser/ui/webui/signin/inline_login_ui.cc | 1 -
 google_apis/gaia/gaia_auth_fetcher.cc             | 4 +++-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/chrome/browser/browser_resources.grd b/chrome/browser/browser_resources.grd
--- a/chrome/browser/browser_resources.grd
+++ b/chrome/browser/browser_resources.grd
@@ -141,7 +141,6 @@
       <include name="IDR_INLINE_LOGIN_HTML" file="resources\inline_login\inline_login.html" flattenhtml="true" allowexternalscript="true" type="BINDATA" preprocess="true" />
       <include name="IDR_INLINE_LOGIN_CSS" file="resources\inline_login\inline_login.css" flattenhtml="true" type="BINDATA" preprocess="true" />
       <include name="IDR_INLINE_LOGIN_JS" file="resources\inline_login\inline_login.js" flattenhtml="true" type="BINDATA" />
-      <include name="IDR_GAIA_AUTH_AUTHENTICATOR_JS" file="resources\gaia_auth_host\authenticator.js" flattenhtml="true" type="BINDATA" />
 
       <include name="IDR_INSPECT_CSS" file="resources\inspect\inspect.css" flattenhtml="true" type="BINDATA" />
       <include name="IDR_INSPECT_HTML" file="resources\inspect\inspect.html" flattenhtml="true" allowexternalscript="true" type="BINDATA" />
diff --git a/chrome/browser/ui/webui/signin/inline_login_ui.cc b/chrome/browser/ui/webui/signin/inline_login_ui.cc
--- a/chrome/browser/ui/webui/signin/inline_login_ui.cc
+++ b/chrome/browser/ui/webui/signin/inline_login_ui.cc
@@ -112,7 +112,6 @@ content::WebUIDataSource* CreateWebUIDataSource() {
   static constexpr webui::ResourcePath kResources[] = {
     {"inline_login.css", IDR_INLINE_LOGIN_CSS},
     {"inline_login.js", IDR_INLINE_LOGIN_JS},
-    {"gaia_auth_host.js", IDR_GAIA_AUTH_AUTHENTICATOR_JS},
 #if defined(OS_CHROMEOS)
     {"edu", IDR_EDU_LOGIN_EDU_LOGIN_HTML},
     {"app.js", IDR_EDU_LOGIN_EDU_LOGIN_JS},
diff --git a/google_apis/gaia/gaia_auth_fetcher.cc b/google_apis/gaia/gaia_auth_fetcher.cc
--- a/google_apis/gaia/gaia_auth_fetcher.cc
+++ b/google_apis/gaia/gaia_auth_fetcher.cc
@@ -39,7 +39,7 @@
 
 namespace {
 
-const size_t kMaxMessageSize = 1024 * 1024;  // 1MB
+//const size_t kMaxMessageSize = 1024 * 1024;  // 1MB
 
 constexpr char kBadAuthenticationError[] = "BadAuthentication";
 constexpr char kBadAuthenticationShortError[] = "badauth";
@@ -295,6 +295,7 @@ void GaiaAuthFetcher::CreateAndStartGaiaFetcher(
     const net::NetworkTrafficAnnotationTag& traffic_annotation) {
   DCHECK(!fetch_pending_) << "Tried to fetch two things at once!";
 
+/*
   auto resource_request = std::make_unique<network::ResourceRequest>();
   resource_request->url = gaia_gurl;
   original_url_ = gaia_gurl;
@@ -353,6 +354,7 @@ void GaiaAuthFetcher::CreateAndStartGaiaFetcher(
       base::BindOnce(&GaiaAuthFetcher::OnURLLoadComplete,
                      base::Unretained(this)),
       kMaxMessageSize);
+*/
 }
 
 // static
-- 
2.17.1

